devicetypes/smartthings/ecobee-thermostat.src/ecobee-thermostat.groovy

```diff
@@ -235,8 +235,8 @@ void setHeatingSetpoint(setpoint) {
 
 	def sendHoldType = holdType ? (holdType=="Temporary")? "nextTransition" : (holdType=="Permanent")? "indefinite" : "indefinite" : "indefinite"
 	if (parent.setHold(heatingValue, coolingValue, deviceId, sendHoldType)) {
-		sendEvent("name":"heatingSetpoint", "value":heatingSetpoint)
-		sendEvent("name":"coolingSetpoint", "value":coolingSetpoint)
+		sendEvent("name":"heatingSetpoint", "value":heatingSetpoint, "unit":location.temperatureScale)
+		sendEvent("name":"coolingSetpoint", "value":coolingSetpoint, "unit":location.temperatureScale)
```

Comment:
I did it mostly because it was consistent with the rest of the references here. Look at lines 233 and 234 just above this.